### PR TITLE
[doc] Update docs for dhcp_relay config cli

### PR DESCRIPTION
**Pull request by yaqiangz**
#### What I did
Updated docs about dhcp_relay config cli

#### How I did it
Updated docs about dhcp_relay config cli

### CODE REVIEW

File: doc/Command-Reference.md [hunk 1]
```diff
@@ -2348,6 +2348,74 @@ This command is used to delete a configured DHCP Relay Destination IP address or
...
+  Restarting DHCP relay service...
+  ```
+
+**config dhcp_relay ipv6 destination add/del**
+
+This command is used to add or del an IPv6 DHCP Relay destination IP address or multiple IP addresses to a VLAN. Note that more than one DHCP Relay Destination IP address can be operated on a VLAN interface.
+
+- Usage:
+  ```
+  config dhcp_relay ipv6 destination (add | del) <vlan_id> <dhcp_destination_ips>
```
Comment (kellyyeh) [inline doc/Command-Reference.md:2391]: Why destination and not helper?

Comment (yaqiangz) [reply doc/Command-Reference.md:2391]: According to our discussion, add ipv4 helper and ipv6 destination. ipv4 helper part I added is in line2351-line2382

File: doc/Command-Reference.md [hunk 2]
```diff
@@ -2348,6 +2348,74 @@ This command is used to delete a configured DHCP Relay Destination IP address or
   Restarting DHCP relay service...
   ```
 
+**config dhcp_relay ipv4 helper add/del**
+
+This command is used to add or delete an IPv4 DHCP Relay helper IP address or multiple IP addresses to a VLAN. Note that more than one DHCP Relay helper IP address can be operated on a VLAN interface.
```
Comment (jcaiMR) [inline doc/Command-Reference.md:2353]: Suggest: This command is used to add or delete IPv4 DHCP Relay helper addresses to a VLAN. ...

File: doc/Command-Reference.md (hunk 1 above)
Comment (jcaiMR) [reply doc/Command-Reference.md:2391]: Yes, helpers are DHCPv4 only concept, DHCPv6 it use relay destination.

File: doc/Command-Reference.md (hunk 2 above)
Comment (yaqiangz) [reply doc/Command-Reference.md]: Thanks for suggesting, fixed

**Review by jcaiMR — APPROVED** [review]

**Review by kellyyeh — APPROVED** [review]

**Pull request merged**